### PR TITLE
Extract XEN chosen node

### PR DESCRIPTION
**Pull request by aanisov**

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/xen-chosen.dtsi [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Device Tree default chosen node for systems running XEN hypervisor
+ *
+ * Copyright (C) 2018 EPAM Systems.
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/xen-chosen.dtsi]: 2019?

Comment (aanisov) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/xen-chosen.dtsi]: yep

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Comment by otyshchenko1** [comment]
lgtm

**Comment by aanisov** [comment]
@iartemenko please merge

**Pull request merged**